### PR TITLE
Bug 1235106 - Add integration test: |lockscreen_notification_tapping_test.js| to test if tapping on notification will unlock/remain lock with different passcodes

### PR DESCRIPTION
**Pull request by snowmantw**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=37115ed579e3cebf7f179b648ae4c3df3f42f8f9

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b2be8cb57d71b4d684923fcdb6f571798491e7d3

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b2be8cb57d71b4d684923fcdb6f571798491e7d3

File: apps/system/test/marionette/lockscreen_notification_tapping_test.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  var fireNotification = function() {
+    var details = {
+     tag: 'test tag',
+     title: 'test title',
+     body: 'test body',
+     dir: 'rtl',
+     lang: 'en'
+    };
+    actions
+      .fireNotification(details);
```
Comment (timdream) [inline apps/system/test/marionette/lockscreen_notification_tapping_test.js]: It is possible to assert that notification "onclick" is invoked here?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5cebdcf8eab88d6d76d0198d810f51e79bf1c497

**Pull request merged**